src/repository.js

```diff
@@ -219,13 +220,15 @@ export class Repository {
     let associations = entity.getMeta().fetch('associations');
 
     for (let property in associations) {
-      let assocMeta = associations[property];
+      if ({}.hasOwnProperty.call(associations, property)) {
```

Comment:
that's http://eslint.org/docs/rules/guard-for-in (together with http://eslint.org/docs/rules/no-prototype-builtins )